backend/src/litigation_api/ingestion/client.py

```
@@ -0,0 +1,109 @@
+import httpx
+from typing import AsyncGenerator, Dict, Any, Optional
+from tenacity import retry, stop_after_attempt, wait_exponential, retry_if_exception_type
+
+logger = logging.getLogger(__name__)
+
+class CRLCAClientError(Exception):
+    """Base exception for CRLCA client errors."""
+    pass
+
+class CRLCAClient:
+    def __init__(self, base_url: str = "https://api.clearinghouse.net/api/v2p1/", timeout: float = 30.0):
```

Ruff (0.15.11)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)